rustler/src/types/map.rs

```diff
@@ -31,12 +31,12 @@ impl<'a> Term<'a> {
     /// ```
     pub fn map_from_arrays(
         env: Env<'a>,
-        keys: &[Term<'a>],
-        values: &[Term<'a>],
+        keys: &[impl Encoder],
+        values: &[impl Encoder],
```

Comment:
Shouldn't this be `impl Encoder + 'a` as @hansihe noted?

Reply:
`Encoder::encode` encodes a Rust value into a term bounds to `env`'s lifetime. So, it actually doesn't need the lifetime bound!